webapp/components/channel_header.jsx

```diff
@@ -362,6 +378,64 @@ export default class ChannelHeader extends React.Component {
                     </ToggleModalButton>
                 </li>
             );
+        } else if (isGroup) {
+            dropdownContents.push(
```

Comment:
Probably but this is how the rest of the code here does it and I'd rather not do a bunch of refactoring for a PR so close to feature complete